packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -863,6 +867,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CC = "";
```

Comment:
This change should be reverted.

Reply:
[8c7547e](https://github.com/facebook/react-native/pull/45114/commits/8c7547ec8823356c1578e79384ec159b1e10c817)